core-web/libs/template-builder/src/lib/components/template-builder/components/dot-layout-properties/dot-layout-properties.component.html

```diff
@@ -8,13 +8,15 @@
         <dot-layout-properties-item
             class="property-item__header"
             [label]="'editpage.layout.properties.header' | dm"
+            [value]="group.get('header').value"
```

Comment:
Why this value?

Reply:
because I need to update the state of the buttons if I change the `header/footer` state from outside the form